providers/mondoo/go.mod

```diff
@@ -128,7 +128,7 @@ require (
 	github.com/sirupsen/logrus v1.9.3 // indirect
 	github.com/skeema/knownhosts v1.3.0 // indirect
 	github.com/spf13/afero v1.11.0 // indirect
-	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/spf13/pflag v1.0.6-0.20201009195203-85dd5c8bc61c // indirect
```

Comment:
Otherwise the tests failed:
```
--> [mondoo] generate CLI json
go: updates to go.mod needed; to update it:
	go mod tidy
make: *** [Makefile:379: providers/build/mondoo] Error 1
Error: Process completed with exit code 2.
```